instrumentation/aws_lambda/lib/opentelemetry/instrumentation/aws_lambda/instrumentation.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsLambda
+      # Instrumentation class that detects and installs the AwsLambda instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        install do |_config|
+          require_dependencies
+        end
+
+        present do
+          # maybe check if ORIG_HANDLER or _HANLDER exist
+          true
+        end
```

Comment:
Could we check to see if the gem is installed? Maybe with `defined?(Aws::Lambda)`? 

With the current code, if we add this to `instrumentation-all`, I think the lambda instrumentation will attempt to install every time.

Reply:
I like that!

Reply:
Thanks, since we are not requiring gem like [aws-sdk-lambda](https://github.com/aws/aws-sdk-ruby/blob/version-3/gems/aws-sdk-lambda/lib/aws-sdk-lambda.rb) , so `Aws::Lambda` won't be presented.